LL/senchas_síl_ébir.ttl

```diff
@@ -655,7 +655,7 @@
     rel:descendantOf <#Dare>.
 
 <#MacCon>
-    a foaf:Person;
+    a irishTitles:Rí;
```

Comment:
I had a quick look.  I think it is referring to Mac Con but I am willing to revise this.